test/k8s/services.go

```diff
@@ -289,7 +289,7 @@ var _ = SkipDescribeIf(helpers.RunsOn54Kernel, "K8sDatapathServicesTest", func()
 
 			It("Tests NodePort with L7 Policy", func() {
 				applyPolicy(kubectl, demoPolicyL7)
-				testNodePort(kubectl, ni, false, false, 0)
+				testNodePort(kubectl, ni, false, true, 0)
```

Comment:
shouldn't these "from-outside" tests be in the `Checks N/S loadbalancing` section below? :thinking: